src/lib/dnssd/Types.h

```diff
@@ -300,31 +300,33 @@ struct ResolvedNodeData
 struct DiscoveredNodeData
 {
     CommonResolutionData resolutionData;
-    CommissionNodeData commissionData;
+    DnssdNodeData nodeData;
+    DiscoveryType nodeType;
 
     void Reset()
     {
         resolutionData.Reset();
-        commissionData.Reset();
+        nodeData.Reset();
+        nodeType = DiscoveryType::kUnknown;
     }
     DiscoveredNodeData() { Reset(); }
 
     void LogDetail() const
     {
         ChipLogDetail(Discovery, "Discovered node:");
         resolutionData.LogDetail();
-        commissionData.LogDetail();
+        nodeData.LogDetail();
     }
 };
 
 /// Callbacks for discovering nodes advertising non-operational status:
 ///   - Commissioners
 ///   - Nodes in commissioning modes over IP (e.g. ethernet devices, devices already
 ///     connected to thread/wifi or devices with a commissioning window open)
-class CommissioningResolveDelegate
+class DiscoverNodeDelegate
 {
```

Comment:
Also, the comments here no longer match the code...

Reply:
As requested this PR does not have the functional changes of operational discovery support so comment is not updated here. However, I have updated the comment for the delegate in the functional PR.